src/compiler/output-targets/dist-custom-elements/index.ts

```diff
@@ -28,11 +32,12 @@ export const outputCustomElements = async (config: d.Config, compilerCtx: d.Comp
     return;
   }
 
-  const timespan = buildCtx.createTimeSpan(`generate custom elements started`);
+  const bundlingEventMessage = 'generate custom elements started';
+  const timespan = buildCtx.createTimeSpan(bundlingEventMessage);
 
   await Promise.all(outputTargets.map((o) => bundleCustomElements(config, compilerCtx, buildCtx, o)));
 
-  timespan.finish(`generate custom elements finished`);
+  timespan.finish(bundlingEventMessage);
```

Comment:
Should this be similar to this? https://github.com/ionic-team/stencil/pull/3078/files#diff-db8331cc26cc3aca3d77bb2dbd81e8f595b9f7e669cfa9d083b9b54b3e57820fR40